utils/_context/_scenarios.py

```diff
@@ -1221,9 +1221,9 @@ class scenarios:
     onboarding_host_container_auto_install = OnBoardingScenario("ONBOARDING_HOST_CONTAINER_AUTO_INSTALL", doc="")
     onboarding_container_auto_install = OnBoardingScenario("ONBOARDING_CONTAINER_AUTO_INSTALL", doc="")
 
-    debugger_method_probes_status = EndToEndScenario(
-        "DEBUGGER_METHOD_PROBES_STATUS",
```

Comment:
Is this scenario deployed elsewhere ?

Reply:
No, in our CI we are using `./run.sh DEBUGGER_SCENARIOS` and running all available tests